mica-webapp/src/main/resources/i18n/en.json

```diff
@@ -1085,6 +1085,16 @@
     }
   },
   "file": {
+    "errors" : {
+      "UPLOAD_ERR_INI_SIZE" : "The uploaded file exceeds the upload_max_filesize directive in php.ini.",
+      "UPLOAD_ERR_FORM_SIZE" : "The uploaded file exceeds the MAX_FILE_SIZE directive that was specified in the HTML form.",
+      "UPLOAD_ERR_PARTIAL" : "The uploaded file was only partially uploaded.",
+      "UPLOAD_ERR_NO_FILE" : "No file was uploaded.",
+      "UPLOAD_ERR_NO_TMP_DIR" : "Missing a temporary folder. Introduced in PHP 5.0.3.",
+      "UPLOAD_ERR_CANT_WRITE" : "Failed to write file to disk. Introduced in PHP 5.1.0.",
+      "UPLOAD_ERR_EXTENSION" : "A PHP extension stopped the file upload.",
```

Comment:
If php didn't received file, show error on php side.
"The uploaded file exceeds the upload_max_filesize directive in php.ini." => in this case (like other cases), fix this problem on drupal side. in this case, file is rejected, don't send it to mica.